.github/workflows/fesom2.1.yml

```diff
@@ -19,10 +19,17 @@ jobs:
     # NK: this changes working directory to fesom2
     - uses: actions/checkout@v2
     
-    - name: Compile model
+    - name: Compile model (binary)
       run: |
         bash -l configure.sh ubuntu
-    
+
+    - name: Compile model (library)
+      run: |
+        bash ./test/ifs_interface/configure_lib.sh -l
+    - name: Library exists
+      run: |
+        bash ./test/ifs_interface/check_exist.sh
```

Comment:
Thomas, did you check it it fails when library is not created?

Reply:
I am returning exit code 0 in that case, so I was assuming this will be working.  From my local Docker, I did not know how to trigger the ./github/workflow

Reply:
![image](https://user-images.githubusercontent.com/26135424/141657442-336e77b5-afe4-438a-ba89-1e350b228a8b.png)